docs/source/WritingTests.rst

```diff
@@ -208,6 +208,10 @@ to be accessed from up to three sources:
    ``MyTest.test_foo``, with variant ``debug-ffff``, the data directory path
    will be ``/home/user/test.py.data/MyTest.test_foo/debug-ffff/``.
 
+.. note:: Unlike INSTRUMENTED tests the ``SIMPLE`` tests only define `file`
```

Comment:
There's an inconsistency here in how you format test types (without double back quotes for INSTRUMENTED, and with them for SIMPLE.

Reply:
Also, it sounds strange to me (remember, not native English speaker) to use "the SIMPLE tests", when the previous test referenced (INSTRUMENTED) did not have an definite article preceding them.  I'd write that as: "Unlike INSTRUMENTED tests, SIMPLE tests only define..."

Reply:
Yep, looks better...